File: 0027-remove-element/0027-remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        count=0
        arr=[]
        length=len(nums)
        for i in range(length):
            if nums[i]==val:
                count+=1
                continue
            else:
                arr.append(nums[i])
        length2=len(arr)
        for i in range(length2):
            nums[i]=arr[i]

        return length-count

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count=0;
        int length=nums.size();
        int index=0;
        for(int i : nums){
            if(i!=val){
                count++;
                nums[index]=i;
                index++;
            }
        }
        return count;
    }
};